pkg/providers/instancetype/suite_test.go

```diff
@@ -137,6 +137,38 @@ var _ = Describe("InstanceTypeProvider", func() {
 							Zone: "test-zone-1c",
 						},
 					},
+					AMIs: []v1beta1.AMI{
+						{
+							ID: "ami-test1",
+							Requirements: []v1.NodeSelectorRequirement{
+								{Key: v1.LabelArchStable, Operator: v1.NodeSelectorOpIn, Values: []string{corev1beta1.ArchitectureAmd64}},
+								{Key: v1beta1.LabelInstanceGPUCount, Operator: v1.NodeSelectorOpDoesNotExist},
+								{Key: v1beta1.LabelInstanceAcceleratorCount, Operator: v1.NodeSelectorOpDoesNotExist},
+							},
+						},
+						{
+							ID: "ami-test2",
+							Requirements: []v1.NodeSelectorRequirement{
+								{Key: v1.LabelArchStable, Operator: v1.NodeSelectorOpIn, Values: []string{corev1beta1.ArchitectureAmd64}},
+								{Key: v1beta1.LabelInstanceGPUCount, Operator: v1.NodeSelectorOpExists},
+							},
+						},
+						{
+							ID: "ami-test3",
+							Requirements: []v1.NodeSelectorRequirement{
+								v1.NodeSelectorRequirement{Key: v1.LabelArchStable, Operator: v1.NodeSelectorOpIn, Values: []string{corev1beta1.ArchitectureAmd64}},
```

Comment:
This should be Arm64. Looks like these were copied across files, so can we make sure that this is correct everywhere it's used?

Reply:
This is another reason why it might make sense to standardize this in the `test.EC2NodeClass`

Reply:
Yeah, I agree now. I will move the status into a central place. This is meant to mock the default AL2 AMI, Why do you believe this should be Arm64?